src/components/NavBar/NavBarSection.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div class="navbar_section">
-		<span class="navbar_section_header">
+		<span class="navbar_section_header">			
```

Comment:
Additional whitespace should be removed.
```suggestion
		<span class="navbar_section_header">
```